src/hotspot/share/opto/vectornode.cpp

```diff
@@ -551,6 +551,7 @@ void VectorNode::vector_operands(Node* n, uint* start, uint* end) {
   case Op_LShiftI:  case Op_LShiftL:
   case Op_RShiftI:  case Op_RShiftL:
   case Op_URShiftI: case Op_URShiftL:
+  case Op_RoundDoubleMode:
```

Comment:
Can you also add a check for Op_RotateRight and Op_RotateLeft here, its count argument (second operand) is treated as a vector/scalar based on following condition
https://github.com/openjdk/jdk/blob/master/src/hotspot/share/opto/superword.cpp#L2609.

Reply:
Thanks for comments. I added those opcodes with a condition in latest commit.